models/public/forward-tacotron/description/forward-tacotron.md

```diff
@@ -0,0 +1,127 @@
+# forward-tacotron (composite)
+
+## Use Case and High-Level Description
+
+ForwardTacotron is a model for the text-to-speech task originally trained in PyTorch\*
```

Comment:
could you please specify what dataset the model was trained on?

Reply:
done